bootstrap.sh

```diff
@@ -138,6 +140,28 @@ install_nodejs() {
   log "✅ Yarn installed"
 }
 
+install_python37_pyenv() {
+  log "⚠️  Installing Python ${PYTHON_VERSION} via pyenv"
+
+  # shellcheck disable=SC2016
+  append_to_dotfiles 'if command -v pyenv 1>/dev/null 2>&1; then\n  eval "$(pyenv init -)"\nfi'
```

Comment:
I'm getting errors with the newlines...

Unlike the pyenv docs, `append_to_dotfiles` doesnt run through `echo -e`, which would convert `\n` to newlines, but I think a one-liner like this would work:
```suggestion
  append_to_dotfiles 'if command -v pyenv 1>/dev/null 2>&1; then eval "$(pyenv init -)"; fi'
```

Reply:
Also looks like pyenv just released 2.0.0, which has a breaking changing for the init: https://github.com/pyenv/pyenv/blob/master/CHANGELOG.md#breaking-changes


Docs are a little confusing because there is an exception for homebrew, but ultimately just using `init --path` instead of `init -` worked for me... so the final line would be:
```suggestion
  append_to_dotfiles 'if command -v pyenv 1>/dev/null 2>&1; then eval "$(pyenv init --path)"; fi'
```